libs/mel/src/lib/pagination-buttons/pagination-buttons.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="relative">
+  <div class="flex flex-row items-center">
+    <gn-ui-button
+      type="outline"
+      [disabled]="currentPage === 1"
+      (buttonClick)="previousPage()"
+      extraClass="pagination-btn-nav mr-14"
+    >
+      <mat-icon class="pagination-btn-arrow material-symbols-outlined"
+        >chevron_left</mat-icon
+      >
+      <span class="font-bold ml-1" translate
+        >mel.datahub.search.pagination.previous</span
+      >
+    </gn-ui-button>
+    <ng-container *ngFor="let page of visiblePages">
+      <ng-container *ngIf="page === '...'">
+        <span class="mr-14">{{ page }}</span>
+      </ng-container>
+      <ng-container *ngIf="page !== '...'">
+        <gn-ui-button
+          [type]="page === currentPage ? 'outline' : 'light'"
+          [disabled]="page === currentPage"
+          (buttonClick)="changePage(page)"
+          extraClass="pagination-btn"
+          >{{ page }}</gn-ui-button
+        >
+      </ng-container>
+    </ng-container>
+    <gn-ui-button
+      type="outline"
+      [disabled]="currentPage === totalPages"
+      (buttonClick)="nextPage()"
+      extraClass="pagination-btn-nav"
+    >
+      <span class="font-bold mr-1" translate
```

Comment:
Same here with the `font-weight` :) . I would also change the font weight for the icon. Maybe to 200 or 300?

Reply:
The icon already is in 300. So do you mean that it should be lighter or bolder?

Reply:
Ah ok, I did not see that. I think it should be a bit thinner, based on the mockup.